toolkit/offchain/src/plutus_script.rs

```diff
@@ -0,0 +1,70 @@
+use anyhow::{anyhow, Context};
+use cardano_serialization_lib::{Address, LanguageKind, NetworkIdKind, PlutusData};
+use plutus::ToDatum;
+use sidechain_domain::{MainchainAddressHash, PolicyId};
+use uplc::ast::{DeBruijn, Program};
+
+use crate::{csl::*, untyped_plutus::*};
+
+/// Wraps a Plutus script cbor
+pub struct PlutusScript {
```

Comment:
I was not sure about adding our PlutusScript wrapper. I'm not against it. But I think that script should know what version of Plutus it is.

Reply:
Added the plutus version. Yeah, I know this is yet-another-wrapper, but we're using two competing plutus data libraries, none of which offers a great API tbh, so I think it's better to hide that ugliness behind a domain-ish object.